build/helm/keylime/charts/keylime-init/templates/ca-job.yaml

```diff
@@ -45,6 +45,12 @@ spec:
                   key: KEYLIME_CA_PASSWORD
             - name: KEYLIME_SECRETS_CA_PW_NAME
               value: "{{ include "keylime.ca.secret.password" . }}"
+            {{- if .Values.global.configmap.configParams }}
+            {{- range $k, $v := .Values.global.configmap.configParams }}
+            - name: {{ $k }}
+              value: {{ $v }}
+            {{- end }}
+            {{- end}}
```

Comment:
I guess that works, alternatively we could do an `envFrom` and include the config map like here?

https://github.com/keylime/attestation-operator/blob/main/build/helm/keylime/charts/keylime-verifier/templates/statefulset.yaml#L31

Reply:
I tried this, but this is an init job and the configmap is not yet created then.

Reply:
Ah right... makes sense :+1: